### DISCUSSION TITLE
后台密码错误

### DISCUSSION THREAD

**Discussion by myedunote** [AlistGo/alist#306] [Q&A]
查看了数据库后台密码，一模一样地输入，但是提示错误

版本：Windows 2.0.4 Fix2，数据库为Windows2.0.4Fix2

**Comment by myedunote** [comment]
找到错误了，不能在反代域名下输入后台密码，否则会提示错误

**Reply by xhofe** [reply]
正确的反向代理是不会导致密码错误的，需要正确的传递Authorization头。

**Reply by winnie2007** [reply]
Authorization头怎样传递？在哪里修改啊，感谢
